function/pkg/operator/operator.go

```diff
@@ -48,7 +47,9 @@ func applyObject(ctx context.Context, c client.Client, u unstructured.Unstructur
 
 	// If object needs update
 	if objFound && !equal {
-		response.Object["spec"] = u.Object["spec"]
+		response = updateConfigurationObject(response, u)
+		//response.Object["spec"] = u.Object["spec"]
+		//TODO: add copy of metadata/labels and metadata/annotations
```

Comment:
During the offline conversation, we figured out that these comments are not valid anymore

Reply:
Removed